### PR TITLE
[How to? pattern articles] Add record microphone article

### PR DESCRIPTION
**Pull request by beaufortfrancois**
@tomayac Please have a look.

Live previews:

https://deploy-preview-8890--web-dev-staging.netlify.app/patterns/media/
https://deploy-preview-8890--web-dev-staging.netlify.app/patterns/media/microphone-record/

### CODE REVIEW

File: src/site/content/en/patterns/media/microphone-record/assets/body.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      rel="icon"
+      href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>🎙️</text></svg>"
+    />
+    <title>How to record audio from a microphone</title>
```
Comment (tomayac) [inline src/site/content/en/patterns/media/microphone-record/assets/body.html:10]:
```suggestion
    <title>How to record audio from the user's microphone</title>
```

Let's make this symmetric to the screen recording title. Please change here and throughout.

File: src/site/content/en/patterns/media/microphone-record/assets/script.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  startRecordButton.disabled = true;
+  stopRecordButton.disabled = true;
+  log("Your microphone audio is not used anymore.");
+});
+
+startRecordButton.addEventListener("click", async () => {
+  // Prompt the user to choose where to save the recording file.
+  const suggestedName = "microphone-recording.webm";
+  const handle = await window.showSaveFilePicker({ suggestedName });
+  const writable = await handle.createWritable();
```
Comment (tomayac) [inline src/site/content/en/patterns/media/microphone-record/assets/script.js:32]: We need the same API support blurb here that we have in the other article.

File: src/site/content/en/patterns/media/microphone-record/demo.njk [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+        display: block;
+        margin-bottom: 4px;
+      }
+      video {
+        display: block;
+        margin-top: 10px;
+        max-width: 100%;
+        background: black;
+        max-height: 480px;
+      }
```
Comment (tomayac) [inline src/site/content/en/patterns/media/microphone-record/demo.njk:43]:
```suggestion
```

Nit: not used here.

**Review by tomayac — APPROVED** [review]
LGTM, with just very few nits.

**Comment by beaufortfrancois** [comment]
@tomayac Thanks!

**Pull request merged**